### DISCUSSION TITLE
Should we switch to a fresh nirsoft bucket?

### DISCUSSION THREAD

**Discussion by rasa** [ScoopInstaller/Scoop#5293] [General]
https://github.com/MCOfficer/scoop-nirsoft is regularly updated and has new apps like [batteryhistoryview](https://blog.nirsoft.net/2022/09/22/view-the-battery-history-of-your-laptop-with-the-batteryhistoryview-tool/) and [appresourcesusageview](https://www.nirsoft.net/utils/app_resources_usage_view.html). Its [update script](https://github.com/MCOfficer/scoop-nirsoft/blob/master/update.py) finds new apps as they are published.

Our [current](https://github.com/ScoopInstaller/Scoop/blob/f93028001fbe5c78cc41f59e3814d2ac8e595724/buckets.json#L5) nirsoft bucket, https://github.com/kodybrown/scoop-nirsoft has not been updated since June 21, 2022. 

Should we switch over?

I [looped in](https://github.com/kodybrown/scoop-nirsoft/issues/184) @kodybrown to get their thoughts.

**Comment by rashil2000** [comment]
Yes, I think we should.

**Reply by rashil2000** [reply]
We would need to ask @MCOfficer as well :)

**Reply by rasa** [reply]
@MCOfficer would [prefer](https://github.com/MCOfficer/scoop-nirsoft/issues/14#issuecomment-1356847087) we move the repo under the ScoopInstaller org. @ScoopInstaller/maintainers Is that ok with us? We should wire it up to get updated via a github action.

**Reply by niheaven** [reply]
That's okay for me.

**Reply by rashil2000** [reply]
I'm okay with that too, as long as we have a dedicated maintainer for it. We are now facing a problem of rarely-maintained official buckets (see the number of issues and PRs in Extras and Nonportable, for example).

Number of official buckets is increasing, but number of maintainers is not 😅

**Reply by niheaven** [reply]
Since too many `manifest add`, and lots of them are large ones...

Okay, that means scoop is being more and more popular, perfect! But we need a automatic installable test CI.

**Comment by rasa** [comment]
So we didn't get a responce from @kodybrown and it's been three weeks, so unless we hear some detractors speak up, I'd like to move forward. I will ask @MCOfficer to move their repo to our org.

**Reply by pureby** [reply]
![futurama-bender](https://user-images.githubusercontent.com/6698649/211804201-fcb7a535-f69d-4d55-8632-7b1cf5d762fe.gif)

**Reply by cvladan** [reply]
News about this? Please act on it.

**Reply by MCOfficer** [reply]
https://github.com/ScoopInstaller/Nirsoft/issues/14

not official yet, but moved to the scoop org

**Comment by kodybrown** [comment]
Hey all. I know this is ancient news now.
I haven't been getting any notifications from GitHub since last August apparently. I feel really bad about that actually. 

It looks like the ScoopInstaller/NirSoft repo has been made the "known" repo, right? That is great!

I actually just created a scheduled task on my computer to monitor NirSoft changes.
Let me know if I can help with the new repo.

**Comment by elisimpson** [comment]
When is ScoopInstaller/NirSoft going to become the official known bucket? It's been a several months now...

**Reply by rasa** [reply]
@elisimpson It's scheduled for the [next release](https://github.com/ScoopInstaller/Scoop/pull/5424).
